Source/MillicastPublisher/Private/Components/MillicastPublisherComponent.cpp

```diff
@@ -37,6 +38,13 @@ UMillicastPublisherComponent::UMillicastPublisherComponent(const FObjectInitiali
 	WS = nullptr;
 	bIsPublishing = false;
 
+	Bitrates = MakeShared<webrtc::PeerConnectionInterface::BitrateParameters>();
+	
+	// Default bitrates
+	Bitrates->max_bitrate_bps = 5'000'000; // 5 megabit (it will ramp to this if it network permits)
```

Comment:
We may want to reduce the default max bitrate; I think @murillo128 suggested 3Mbps? Whatever is decided, we should make sure this change is in the release notes so that users are aware of the change, as it has a cost impact.

Reply:
Okay this is now set to 2.5 megabits, which was what it was using under the hood when it was unconfigured: https://github.com/millicast/millicast-publisher-unreal-engine-plugin/pull/6/commits/e64c5c2286c77e19c4f8b27ecc36ee3096a39d26